client/src/components/pages/views/ViewAllVolunteers.js

```diff
@@ -20,7 +21,14 @@ export class ViewAllVolunteers extends Component {
                 firstName: volunteer.firstName,
                 lastName: volunteer.lastName,
                 email: volunteer.email,
-                key: volunteer._id
+                key: volunteer._id,
+                //on click needs
+                viewProfile: (
+                    <Link to={'/volunteer/' + volunteer._id} className="button large primary">
+                        {' '}
```

Comment:
same as above, why are there these spaces here? it seems kinda ugly, especially having them on these two lines like this

Reply:
Prettier added them on save, I dont believe refactoring with prettier formatting turned off would be worth the time to remove...